database/rrdhost.c

```diff
@@ -734,7 +746,45 @@ struct label *load_kubernetes_labels()
     l = add_label_to_list(l, "beta", "Beta value from k8s", LABEL_SOURCE_KUBERNETES);
     l = add_label_to_list(l, "_os_version", "os version from k8s", LABEL_SOURCE_KUBERNETES);
     return l; */
-    return NULL;
+
+    struct label *l=NULL;
+    char *label_script = mallocz(sizeof(char) * (strlen(netdata_configured_primary_plugins_dir) + strlen("get-kubernetes-labels.sh") + 2));
+    sprintf(label_script, "%s/%s", netdata_configured_primary_plugins_dir, "get-kubernetes-labels.sh");
```

Comment:
Can we not get a string constant out of autotools at build-time to avoid the malloc/free ?